app/helpers/application_helper.rb

```diff
@@ -568,46 +568,57 @@ def model_report_type(model)
     nil
   end
 
+  def show_taskbar_in_header?
+    return false if @explorer
+    return false if controller.action_name.end_with?("tagging_edit")
+
+    hide_actions = %w(
```

Comment:
Hey, you're completely right, I even had it as a constant at some point.

Except.. it felt like looking for a variable in a 40 line long function may be a bit easier than looking for a constant in a 1700 line module :).

But agreed that we may want to pull it out (together with the other toolbar related methods) to ..say, `app/helpers/toolbar_helper.rb` (which is included from there anyway) .. and then it makes *complete* sense to have it as a constant.

So.. yeah, I was lazy, adding to my list :) (unless you want to take it ;))